site/layouts/shortcodes/js-dismiss.html

```diff
@@ -2,14 +2,14 @@
 
 {{- $name := .Get 0 -}}
 
-Dismissal can be achieved with the `data` attribute on a button **within the {{ $name }}** as demonstrated below:
+Dismissal can be achieved with the `data-bs-dismiss` attribute on a button **within the {{ $name }}** as demonstrated below:
 
 ```html
 <button type="button" class="btn-close" data-bs-dismiss="{{ $name }}" aria-label="Close"></button>
 ```
 
-or on a button **outside the {{ $name }}** using the `data-bs-target` as demonstrated below:
+or on a button **outside the {{ $name }}** using the additional `data-bs-target` as demonstrated below:
 
 ```html
-<button type="button" class="btn-close" data-bs-dismiss="{{ $name }}" data-bs-target="#my-{{ $name }}" aria-label="Close"></button>
+<button type="button" class="btn btn-primary" data-bs-dismiss="{{ $name }}" data-bs-target="#my-{{ $name }}" aria-label="Close"></button>
```

Comment:
Not reviewing the entire PR but regarding this modification, for a better understanding, I'd stay with the same classes between the 2 examples.

```suggestion
<button type="button" class="btn-close" data-bs-dismiss="{{ $name }}" data-bs-target="#my-{{ $name }}" aria-label="Close"></button>
```